uui/components/pickers/PickerTogglerTag.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as types from '../types';
+import { Tag, TagProps } from '../widgets';
+import { DataRowProps } from '@epam/uui-core';
+
+export interface PickerTogglerTagProps<TItem, TId> extends TagProps {
+    /** Defines component size */
+    size?: types.ControlSize;
+    /** If this is true, then the PickerTogglerTag will be an additional tag with the number of collapsed elements in the caption. */
```

Comment:
/** Indicates that tag is collapsed rest selected items, like '+N items selected' */

Reply:
done